### ISSUE TITLE
TypeError exception with Anaconda's Python 3.9.5

### ISSUE THREAD

**Issue by ahernandezmiro** [hsahovic/poke-env#316]
Due to incompatibilities between wsl and keras/tensorflow I am trying to run everything under Anaconda. However, the following exception appears on any execution:
`...
 class EnvPlayer(Player, Env, ABC):  # pyre-ignore
TypeError: metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases
`

Kind of weird since it was not happening when running basic players from inside wsl. Python version is 3.7.3 in wsl

Is this really a python version issue or can this be fixed changing EnvPlayer class? I'm not into gym-env development but it looks like a simple inheritance problem.

**Comment by hsahovic** [comment]
Hey @ahernandezmiro! Did you install `poke-env` through `pip` or via `git`?

Can you try `pip install gym==0.24.1`? The version that poke-env requires might have been overwritten.

**Comment by ahernandezmiro** [comment]
Hi @hsahovic I installed poke-env through pip in the anaconda environment and I think I did the same for wsl.
Can confirm that this issue has been solved after reinstalling the mentioned gym package.
Now I see the following error
`from poke_env.player.openai_api import ObservationType
ModuleNotFoundError: No module named 'poke_env.player.openai_api'`
Should I close this issue and open a new one or can we follow it here?
